### PR TITLE
br tags in some html files correctly closed.

### LINKED ISSUE #158: [Bug] Incorrect Closing of `br` tags in some html files
## Description
In the html files below, the br tag is wrongly closed there by preventing prettier from formatting the code.

in docs folder:
1. index.html

In specimen > contents:
1. archive-page
2. blog-index
3. blog-post
4. default-page
5. home-index
6. license-page
7. person-page
8. program-index
9. search-index
10. team-index
11. walkthrough-page

in tests folder:
1. index-logo
2. index.html
3. product-logo

The incorrect way the br tag is written prevents prettier from formatting the code.

1.The br tag should be correctly closed, this will enable prettier work without throwing error(s).

- [ ] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by SisiVero**
## Fixes
- Fixes #158  by @SisiVero

## Description
This Pull Request addresses the issues of wrongly closed tags in the html files below:
In docs folder:
index.html

In specimen > contents:
archive-page
blog-index
blog-post
default-page
home-index
license-page
person-page
program-index
search-index
team-index
walkthrough-page

In tests folder:
index-logo
product-logo

## Tests
1. The br tags were correctly closed in the affected html files.
2. Prettier formats the code now.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

### CODE REVIEW

File: specimen/contexts/archive-page.html [hunk 1]
```diff
@@ -1,300 +1,473 @@
 <!DOCTYPE html>
 <html lang="en-US">
-<head>
-<title>Archive Page - Context</title>
+  <head>
```
Comment (possumbilities) [inline specimen/contexts/archive-page.html]: there are changes here unrelated to correcting the `br` tag. Please revert

**Comment by possumbilities** [comment]
Hi @SisiVero It looks like you made a number of formatting changes with prettier at the moment. We try not to do formatting changes with other PRs. 

There's already an issue #5 that would require solidification before we do major formatting shifts (even automatically), but we want to get there!

I'd suggest reverting the changes to only what is outlined in the Issue.

**Comment by SisiVero** [comment]
> Hi @SisiVero It looks like you made a number of formatting changes with prettier at the moment. We try not to do formatting changes with other PRs.
> 
> There's already an issue #5 that would require solidification before we do major formatting shifts (even automatically), but we want to get there!
> 
> I'd suggest reverting the changes to only what is outlined in the Issue.

Hi @possumbilities , noted. 
I'll do so asap.
Thank you.

**Comment by SisiVero** [comment]
Hi @possumbilities , I've reverted the changes and maintained the ones mentioned in the issue. Do confirm, thank you.

**Comment by possumbilities** [comment]
@SisiVero great work so far! I believe the files within the `/specimen/tests` directory should also have this change applied.

**Comment by SisiVero** [comment]
Thank you @possumbilities, I'll check and apply accordingly.

**Comment by SisiVero** [comment]
Hi @possumbilities , I've applied the changes in the mentioned folder files. Do check and confirm.
Thank you.

**Comment by possumbilities** [comment]
Hi @SisiVero I'm still not seeing the relevant changes in the `specimen/tests` directory. There are three `.html` files within that directory that would need to be included in this PR for it to move forward.

Please revisit, and correct.

**Comment by SisiVero** [comment]
Hi @possumbilities , I'll check and effect immediately. 
Thank you.

**Comment by SisiVero** [comment]
Hi @possumbilities , change to the specified folder has been effected, do check and confirm, thank you.

**Review by possumbilities — DISMISSED** [review]
~Wonderful! Thank you. Approved.~

**Comment by possumbilities** [comment]
Hi @SisiVero you need to update your branch with changes from `main` so that they checks pass and the preview loads correctly. 

Thanks

**Comment by possumbilities** [comment]
@SisiVero I see all the files are there now.

One last thing is you need to update your branch with the most recent changes in `main`.

**Comment by SisiVero** [comment]
Hi @possumbilities , I'll do so now.

**Comment by SisiVero** [comment]
Hi @possumbilities , I've merged changes from main to this branch, could you check to see if there are conflicts?

**Comment by possumbilities** [comment]
@SisiVero Did you also make sure to update `main` before you merged? (if not, you'll need to update your local `main` and then merge those into your branch)

File: netlify-deploy.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    cp -a src/* docs/vocabulary/
+    echo 'done.'
+    echo
+}
+
+
+#### MAIN #####################################################################
+
+remove_previous_from_docs
+publish_to_docs
```
Comment (possumbilities) [inline netlify-deploy.sh]: You shouldn't need to include this file in this PR.

**Comment by SisiVero** [comment]
Hi @possumbilities , I have no idea why the checks are still failing. Initially there were conflict because my branch is not up to date even after I pulled. I was able to resolve this by going to my forked repo and syncing. This updated new changes, I then pull again from my terminal and then merged from main to my branch and everything is up to date right now. I can see new changes made to the main branch few hours ago.

**Comment by SisiVero** [comment]
> @SisiVero Did you also make sure to update `main` before you merged? (if not, you'll need to update your local `main` and then merge those into your branch)

@possumbilities , I have done this and seen the new changes.

File: netlify-deploy.sh
Comment (possumbilities) [inline netlify-deploy.sh:1]: this PR would delete this file which needs to stay put, unchanged.

**Comment by possumbilities** [comment]
@SisiVero I'll try to look into it further tomorrow. It looks like something went wrong with your upstream merger from `main`.

File: netlify-deploy.sh
Comment (SisiVero) [reply netlify-deploy.sh:1]: On it

**Comment by SisiVero** [comment]
> @SisiVero I'll try to look into it further tomorrow. It looks like something went wrong with your upstream merger from `main`.

okay @possumbilities , thank you. I'll check too from my end tomorrow. Goodnight.

**Pull request closed without merging**